packages/client/src/index.ts

```diff
@@ -27,16 +27,19 @@ export type {
   OsdkActionParameters,
   OsdkObject,
   PageResult,
+  Result,
```

Comment:
If someone uses `fetchPageWithError` the type is wrapped with `Result`, so thought it best to expose here